tests/dashboard/components/file/test_views.py

```diff
@@ -0,0 +1,405 @@
+import json
+import uuid
+from unittest import mock
+
+import elasticSearchFunctions
+import pytest
+from components import helpers
+from django.urls import reverse
+from main import models
+
+
+@pytest.fixture
+def dashboard_uuid(db):
+    helpers.set_setting("dashboard_uuid", str(uuid.uuid4()))
+
+
+@pytest.fixture
+def transfer(db):
+    return models.Transfer.objects.create()
+
+
+@pytest.fixture
+def sip(db):
+    return models.SIP.objects.create()
+
+
+@pytest.fixture
+def file(db):
+    return models.File.objects.create()
+
+
+def test_file_details(admin_client, dashboard_uuid, file, mocker):
```

Comment:
Instead of using the `mocker` fixture which needs an extra dependency, let's start moving towards Python's standard [`unittest.mock`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch) module.

Reply:
Could we replace `mocker` entirely with `unittest.mock`?

Reply:
sure